Azure-Samples/azure-intelligent-edge-patterns
# Adding kubeflow-on-azure section

## Purpose

* Adding section about Kubeflow. With brief review or GPU-enabled containers, Kubernetes, and step-by-step instructions how to install Kubeflow on Azure.

## Does this introduce a breaking change?

```
[ ] Yes
[ x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
[follow instructions see that it results in functioning Kubeflow]
```

* Test the code

```
Manual testing. Follow instructions see that it results in a functioning Kubeflow instance.
```

## What to Check
Verify that the following are valid
* You should be able to create: NVidia GPU docker container, Kubernetes cluster, Kubeflow instance.

## Other Information
